Test coverage #44

g82005 opened the pull request:
Test coverage configuration is set up now with codecov & jacoco.
- Some of the paths are ignored for now, since we have not work on these classes.
- To run test coverage report locally, enter "gradle createDebugAndroidTestCoverageReport" in termical in Android Studio. The report will be generated in "app/build/reports/coverage".
- badges for travis and codecov will show up in readme file.

g82005 committed:
Test coverage configuration #44

Setting up test coverage configuration

g82005 committed:
Modify yml file #44

Updating yml trying to get the coverage report unploaded

g82005 committed:
Modtify yml for test coverage configuration #44

Modtify yml for test coverage configuration : adding script under stage.

g82005 committed:
Modtify yml for test coverage configuration #44

Modtify yml for test coverage configuration : changin script under stage.

g82005 committed:
Test coverage configuration #44

Adding codecov.yml

g82005 committed:
codecov configuration #44

codecov configuration

g82005 committed:
codecov configuration #44

2nd time modify codecov file

g82005 committed:
Update codecov.yml #44

Update codecov.yml correcting ignore paths

g82005 committed:
Update codecov.yml #44

Modifying codecov.yml

g82005 committed:
Adding badges for travis and codecov #44

Adding badges for travis and codecov

soenAnt commented:
There are conflicts to be resolved, Cannot merge automatically. Please take a look and verify those conflicts.

Thank you

soenAnt reviewed (changes requested):
Please verify the discrepancies before attempting a PR - thank you

Keeran10 commented on app/src/androidTest/java/SettingsActivityTest.java:
```diff
@@ -17,13 +17,13 @@
     public ActivityTestRule<SettingsActivity> settingsActivityActivityTestRule =
             new ActivityTestRule<>(SettingsActivity.class);
 
-    @Test
```
Test class SettingsActivity has been removed from the project. Moving forward, tests classes will mention the feature they test. This is one of the source of conflicts. Also, gradle file should be easy to fix.

g82005 replied:
Gradle file onflicts resolved.

Keeran10 commented:
Can you post a screenshot of the app/build/reports/coverage on this pr to confirm implementation? Thanks.

g82005 committed:
Resovling conflits on test coverage #44

Resovling conflits on test coverage

g82005 committed:
Merge branch 'Dev' into Test_Coverage

g82005 commented:
> Please verify the discrepancies before attempting a PR - thank you

It is resolved thanks for reminding

g82005 commented:
![image](https://user-images.githubusercontent.com/43104118/53760543-26952780-3e91-11e9-906c-280e581d4dc7.png)
test coverage for this branch so far

g82005 committed:
fixing travis failure for test coverage #44

Failed travis ci since last time resovling conflicts, changing configuration and try to tix failure for test coverage

g82005 committed:
fixing travis failure for test coverage #44

 Configuration changes fixing travis failure for test coverage

g82005 committed:
travis configuration fixing build fails #44

changing minsdkversion to 23 and emulator version to 23

g82005 committed:
travis configuration fixing build errors #44

changing the emulator version to 24

g82005 commented:
The issure can not be solved, creating new branch and implementing codecov there.
